src/core/globals.hpp

```diff
@@ -387,6 +387,12 @@ namespace big
 			NLOHMANN_DEFINE_TYPE_INTRUSIVE(spawn_ped, preview_ped)
 		} spawn_ped{};
 
+		struct train
+		{
+			bool derail_train = false;
+			bool drive_train = false;
+		} train{};
```

Comment:
Just a reminder, you can use NLOHMANN_DEFINE_TYPE_INTRUSIVE to save settings, the code looks good to me

Reply:
the idea with them though was they wouldnt be saved

Reply:
ah, alright

Reply:
:)